backend/installfinders/launchers/steam/steam.go

```diff
@@ -99,8 +100,16 @@ func FindInstallationsSteam(steamPath string, launcher string, platform common.L
 
 			var branch common.GameBranch
 			userConfig := manifest["AppState"].(map[string]interface{})["UserConfig"].(map[string]interface{})
-			betakey, ok := userConfig["betakey"]
-			if !ok {
+
+			var betakey string
+			for k, v := range userConfig {
+				if strings.EqualFold(k, "BetaKey") {
+					betakey = v.(string)
+					break
+				}
+			}
+
+			if betakey == "" || betakey == "stable" {
```

Comment:
```suggestion
			if betakey == "" || betakey == "public" {
```
I have no idea how I got this wrong in the previous suggestion. It's working with `"public"` in my local build right now.